src/patternfly/components/Table/table-grid.scss

```diff
@@ -410,6 +411,13 @@
     padding-right: 0;
   }
 
+  .pf-c-table__button {
+    --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-grid--cell--PaddingTop);
```

Comment:
I think you'll need to change this var name

Reply:
The plan is to update the old vars to this pattern: https://github.com/patternfly/patternfly/issues/2983